### ISSUE TITLE
Module console return value does not work

### ISSUE THREAD

**Issue by Toxantron** [PHOENIXCONTACT/MORYX-MaintenanceWeb#12]
**Describe the bug**
When invoking console methods from the maintenance, the return value is no longer shown

**To Reproduce**
1. Open Maintenance
2. Invoke any console method with a return type other than void
3. You will only see an empty result

**Expected behavior**
I would expect to see the methods return value in Entry Format

**Additional context**
*-*

**Comment by Toxantron** [comment]
@mmarkwort would you mind taking a look at this?

**Comment by mmarkwort** [comment]
After some investigation I only found a problem when just a simple type (like e.g. string) is returned. I tested it with the `TestModule`: 

* `Echo` did not work
* `ServerInfo` delivered a valid result

Simple types will only generate an `Entry` with an `EntryValue` but with no `SubEntries`. This is currently not supported by the Maintenance.

I fixed the issue with simple types (see https://github.com/PHOENIXCONTACT/MORYX-MaintenanceWeb/pull/17). We should discuss if the class `MethodEntry` also should describe the return type. This would help to select the right editor.
